weldx/tests/asdf_tests/test_asdf_validators.py

```diff
@@ -163,6 +164,8 @@ def test_shape_validation_error_exception(shape, exp, err):
         assert _val(shape, exp)
 
 
+@pytest.mark.parametrize("copy_arrays", [True, False])
+@pytest.mark.parametrize("lazy_load", [True, False])
```

Comment:
I have added the different loading parameters here for testing but it seems like that is not the issue
arrays apparently get read as `NDArrayType` @marscher